### PR TITLE
LPS-163793 Portal properties configurations are leaked through Instance settings

### PR DESCRIPTION
**Pull request by drewbrokke**
This is an update for [LPS-163793](https://issues.liferay.com/browse/LPS-163793).

@ChrisKian will you please double-check my work and verify that it still meets the requirements? Thank you!

### CODE REVIEW

File: modules/apps/server/server-admin-web/src/main/resources/META-INF/resources/mail_fields.jsp [hunk 1]
```diff
@@ -23,22 +23,25 @@ PortletPreferences companyPortletPreferences = PrefsPropsUtil.getPreferences(com
 
 PortletPreferences systemPortletPreferences = PrefsPropsUtil.getPreferences(0L);
 
-Function<String, String> function = (String key) -> companyPortletPreferences.getValue(key, systemPortletPreferences.getValue(key, PropsUtil.get(key)));
+Function<Boolean, Function<String, String>> basePreferenceFunctionFunction = showPropsValues -> key -> companyPortletPreferences.getValue(key, systemPortletPreferences.getValue(key, showPropsValues ? PropsUtil.get(key) : StringPool.BLANK));
+
+Function<String, String> restrictedPreferenceFunction = basePreferenceFunctionFunction.apply(permissionChecker.isOmniAdmin());
```
Comment (ChrisKian) [inline modules/apps/server/server-admin-web/src/main/resources/META-INF/resources/mail_fields.jsp:28]: The "Admin" in `permissionChecker.isOmniAdmin()` needs to be lower-cased, but aside form that your changes work as expected.

Comment (drewbrokke) [reply modules/apps/server/server-admin-web/src/main/resources/META-INF/resources/mail_fields.jsp:28]: Oof, good catch, thanks Chris.

**Comment by drewbrokke** [comment]
ci:forward

**Comment by drewbrokke** [comment]
ci:forward

**Pull request closed without merging**